pychromecast/controllers/homeassistant.py

```diff
@@ -53,13 +58,13 @@ def __init__(
         #   lovelacePath?: string | number | null;
         #   urlPath?: string | null;
```

Comment:
What are these comments?

Reply:
It's the typescript definition of the message

Reply:
I'd add a comment explaining that.